### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Jamulus app](https://hosted.weblate.org/projects/jamulus/jamulus-app/).

It also includes following components:

* [Jamulus/Windows Installer](https://hosted.weblate.org/projects/jamulus/windows-installer/)

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by ann0see — COMMENTED** [review]
@ignotus666 @dzpex @MarongHappy @henkdegroot @melcon @Snayler @BLumia could you please verify that the translations in this PR are ok?

File: src/translation/translation_zh_CN.ts [hunk 1]
```diff
@@ -119,7 +119,7 @@
     <message>
         <location filename="../util.cpp" line="745"/>
         <source>Norwegian Bokmål</source>
-        <translation>书面挪威语</translation>
+        <translation>노르웨이 보크몰어</translation>
```
Comment (BLumia) [inline src/translation/translation_zh_CN.ts:122]:
```suggestion
        <translation>书面挪威语</translation>
```

I guess this is a translation update that was submitted to the wrong language? The change is Korean but this change was submitted to Chinese language...

**Review by BLumia — CHANGES_REQUESTED** [review]

File: src/translation/translation_zh_CN.ts (hunk 1 above)
Comment (ann0see) [reply src/translation/translation_zh_CN.ts:122]: Can you fix it on Weblate please?

Comment (BLumia) [reply src/translation/translation_zh_CN.ts:122]: Done from the Weblate side.

**Comment by ignotus666** [comment]
@ann0see Spanish is ok.

**Comment by ann0see** [comment]
Thanks @ignotus666 please approve this PR then.

**Review by ignotus666 — APPROVED** [review]

**Comment by henkdegroot** [comment]
@ann0see I am sure I posted a comment but don't see it here. I will have the dutch updated in weblate by 27 march.

**Comment by ann0see** [comment]
You commented on the website :-). Thanks nevertheless.

File: src/translation/translation_zh_CN.ts (hunk 1 above)
Comment (ann0see) [reply src/translation/translation_zh_CN.ts]: Thanks.

**Comment by henkdegroot** [comment]
@ann0see, I have completed the required updates in weblate (what a sloooooow system to use). Hope they appear in the PR sson.

**Comment by ignotus666** [comment]
@henkdegroot bear in mind that you can always do it the "old" way if you prefer, translating locally with your editor of choice and making a PR. I agree that Weblate can be excruciatingly slow.

**Comment by ann0see** [comment]
Any updates here? Is it ok?

**Comment by henkdegroot** [comment]
Dutch is looking okay to me.

**Review by ignotus666 — APPROVED** [review]

**Comment by ann0see** [comment]
Thanks. All other Translators should have a look at their languages too to allow a merge.

**Review by ann0see — APPROVED** [review]

File: src/translation/translation_it_IT.ts [hunk 2]
```diff
@@ -1194,7 +1194,7 @@ E&apos; stato disattivato l&apos;audio del tuo canale ed inserito il &quot;Disat
     <message>
         <location filename="../clientdlgbase.ui" line="343"/>
         <source>Delay</source>
-        <translation>Delay</translation>
+        <translation>Ritardo</translation>
```
Comment (pljones) [inline src/translation/translation_it_IT.ts:1197]: This should express something more like "Aspettare", I think - but I'm not sure of the contextual difference.
* "Se arrivi un po’ in ritardo non succede niente (if you arrive a little late, that’s OK)" from https://contentoitaliano.com/the-italian-verb-succedere-to-happen-to-occur/
* "Il gatto aspetta la cena – the cat is waiting for dinner" from https://contentoitaliano.com/what-does-aspettare-mean-in-italian/

So it's not that clear...  If it's definite that "Ritardo" is more appropriate for wall clock time delay, this is fine.

File: src/translation/translation_nl_NL.ts [hunk 3]
```diff
@@ -2796,7 +2796,7 @@ We hebben uw kanaal gedempt en &apos;Demp mijzelf&apos; geactiveerd. Los eerst h
     <message>
         <location filename="../clientsettingsdlgbase.ui" line="986"/>
         <source>Custom Directories:</source>
-        <translation>Eigen adresboek</translation>
+        <translation>Eigen adresboeken</translation>
```
Comment (pljones) [inline src/translation/translation_nl_NL.ts:2799]: Style question.  We've highlighted key words in Jamulus terminology in English by capitalising them (making proper nouns): have we said the translations should follow the style guide?

Comment (pljones) [reply src/translation/translation_nl_NL.ts:2799]: (Not that the UI follows the guide yet...)

File: src/translation/translation_it_IT.ts (hunk 2 above)
Comment (ignotus666) [reply src/translation/translation_it_IT.ts:1197]: "Ritardo" is fine. It's very similar in Spanish ("retardo"). In your first example, a literal translation would be "if you arrive with a little delay, that's OK".

Comment (pljones) [reply src/translation/translation_it_IT.ts:1197]: "quanto tempo aspetto" is "how long do I wait" - and the "delay", here, is how long you wait before the signal arrives, that's all.  "qual è il ritardo" - "what is the delay" - isn't quite the right meaning.  One is cause (ritardo), one is effect (aspetto).  But again, I'm not going to hold anyone to it as I don't know the language.  (It's just the English is vague.)

Comment (ignotus666) [reply src/translation/translation_it_IT.ts:1197]: You can see a few examples of how ritardare is used for delay [here](https://www.wordreference.com/enit/delay). If you say "qual è il ritardo"  I don't think it has the meaning of "what's causing the delay" but more literally "what is the delay" as in "what is the amount of time elapsing between x and y".

**Review by pljones — APPROVED** [review]
As I've noted, at some point we _may_ want to review the UI strings for conformance to the style guide use of specific terms.

**Comment by ann0see** [comment]
@pljones I believe that can be done on Weblate. I'll regenerate the ts files.

**Pull request merged**